cpp/src/components/connectivity.cu

```diff
@@ -38,7 +38,7 @@ namespace detail {
  * This implementation comes from [1] and solves component labeling problem in
  * parallel on CSR-indexes based upon the vertex degree and adjacency graph.
  *
- * [1] Hawick, K.A et al, 2010. "Parallel graph component labelling with GPUs and CUDA"
+ * [1] Hawick, K.A et al, 2010. "Parallel graph component labelling with GPUs and CUDA_VER"
```

Comment:
This is not a valid change.  `CUDA` appears in the title of the paper being referenced.